.ko.yaml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # We need a shell for a lot of redirection/piping to work
-defaultBaseImage: gcr.io/distroless/base:debug-nonroot
+defaultBaseImage: gcr.io/distroless/base-debian12:debug-nonroot
```

Comment:
`gcr.io/distroless/static-debian12:nonroot` I think should work? We shouldn't need a shell.

Reply:
What about the comment above?

If the shell was only needed when entering the it for debugging, I imagine the comment would mention that instead...

Reply:
good point, lemme test this locally. I’m not sure what is using the shell, any guesses @bobcallaway?

Reply:
@cpanato maybe knows?

Reply:
Using this as the base image, seems like everything is working - I can start the log, upload and fetch an entry. 

Maybe the CLI needs a shell?

Reply:
this is a copypasta job from cosign: https://github.com/sigstore/cosign/commit/8974bdc695ee5ce6ca5356c54101c829d144b23c

Reply:
not relevant for rekor IMO

Reply:
OK, I'll follow up on the Cosign thread to double check if we need the shell. Just updated the base image, PTAL